test/UnitTestFramework/UnitTestFramework.js

```diff
@@ -12,12 +12,14 @@
 //     }
 //   How to define and run tests:
 //     var tests = [ { name: "test name 1", body: function () {} }, ...];
-//     testRunner.run(tests);
+//     testRunner.runTests(tests);       // regular tests
+//     testRunner.runAsyncTests(tests);  // tests loaded with WScript.LoadModule

```

Comment:
Yes, this can be used with any code that is executed asynchronously and not module-only.

---
In reply to: [110754143](https://github.com/Microsoft/ChakraCore/pull/2822#discussion_r110754143) [](ancestors = 110754143)